WordPress/Classes/Services/NotificationSyncService.swift

```diff
@@ -0,0 +1,304 @@
+import Foundation
+
+
+// MARK: - Notifications
+//
+let NotificationSyncServiceDidUpdateNotifications = "NotificationSyncServiceDidUpdateNotifications"
+
+
+// MARK: - NotificationSyncService
+//
+class NotificationSyncService
+{
+    /// Returns the Main Managed Context
+    ///
+    private var contextManager = ContextManager.sharedInstance()
```

Comment:
Not sure what you meant by the `let` comment, but you can definitely bail without setting all the properties:

https://gist.github.com/koke/0c515662acd0ca46f2bfd772edccacee